core-sitemaps.php

```diff
@@ -18,58 +18,58 @@
  * Description:       A feature plugin to integrate basic XML Sitemaps in WordPress Core
  * Author:            Core Sitemaps Plugin Contributors
  * Author URI:        https://github.com/GoogleChromeLabs/wp-sitemaps/graphs/contributors
- * Text Domain:       core-sitemaps
+ * Text Domain:       sitemaps
  * Domain Path:       /languages
  * Requires at least: 5.3
  * Requires PHP:      5.6
  * Version:           0.2.0
  */
 
 // The limit for how many sitemaps to include in an index.
-const CORE_SITEMAPS_MAX_SITEMAPS    = 50000;
```

Comment:
In the  main plugin file, we need some `function_exists` or `class_exists` checks to not load all the classes and functionality below in case they already exist in core.

Reply:
👍

Reply:
See https://github.com/GoogleChromeLabs/wp-sitemaps/pull/182/commits/ce0e9e24dd5110c61b1422624bd9cf695ab91778